projectDocs/issues/triage.md

```diff
@@ -143,10 +143,10 @@ An issue is triaged if it is ready to be worked on.
 New features and enhancements should be [well defined](#new-features-1) before applying the `triaged` label.
 Once a bug has clear steps to reproduce and is well documented, the `triaged` label can be applied.
```

Comment:
This line (and the one following) suggests that community members are fine to triage and prioritize bugs. Is that correct?

Reply:
@XLTechie That's right. Our rubric is:
- Well formed bugs following the issue template, which can easily be understood and isn't missing any important debug information
- Can clearly be reproduced
- Can clearly be prioritised
- Doesn't require a controversial solution